src/components/TransferResult/index.tsx

```diff
@@ -4,8 +4,11 @@ import * as React from 'react';
 import {Link, RouteComponentProps} from 'react-router-dom';
 import {ROUTE_WALLETS} from '../../constants/routes';
 import {STORE_ROOT} from '../../constants/stores';
+import {NumberFormat} from '../NumberFormat';
 import './style.css';
 
+const MAX_ACCURACY = 8;
```

Comment:
May be it make sense to move this constant to the separate file? As I understand max accuracy is equal through the app. The same for DEFAULT_ACCURACY in the src/pages/TransferPage/index.tsx

Reply:
For now this is the only place where we need this constants, i believe we can easily move it when we will now how it should be distributed inside our app, for now it's not clear where this separate file should be. Please let me now your thoughts on this.

Reply:
It makes sense now for me